jdaviz/core/helpers.py

```diff
@@ -265,3 +267,106 @@ def get_model_parameters(self, models=None, model_label=None, x=None, y=None):
                     param_units[key].get(param_name, None))
 
         return parameters_cube
+
+    def show_inline(self):
+        """
+        Display the jdaviz application inline in a notebook.  Note this is
+        functionally equivalent to displaying the cell ``self.app`` in the
+        notebook.
+
+        See Also
+        --------
+        show_in_sidecar
+        show_in_new_tab
+        """
+        display(self.app)
+
+    def show_in_sidecar(self, **kwargs):
+        """
+        Display the jdaviz application in a "sidecar", which by default is a tab
+        on the right side of the jupyterlab  interface.
+
+        Parameters
+        ----------
+        title : str, optional
+            The title of the sidecar tab.  Defaults to the name of the
+            application - e.g. "specviz".
+        anchor : str
+            Where the tab should appear, by default on the right.  To see all
+            options see the ``sidecar.Sidecar.anchor.values``.
+
+        additional keywords are passed into the ``sidecar.Sidecar`` constructor.
+        See ``jupyterlab-sidecar`` for the most up-to-date options.
```

Comment:
Move this before "Parameters". It is not rendering properly in https://jdaviz--952.org.readthedocs.build/en/952/api/jdaviz.core.helpers.ConfigHelper.html#jdaviz.core.helpers.ConfigHelper.show_in_sidecar

Also maybe give a direct link to the said documentation.

Reply:
original motivation was that sidecar does not have a proper docs page to link to... but now that you point this out I can point to the github repo which is the closest thing it has - will fix.